### PR TITLE
Dash/cleanup docs

### PR DESCRIPTION
**Pull request by pawellysy**
Fixed some URL's and indentation error in the *.md files.

### CODE REVIEW

File: docs/dashboards/types-of-components.md [hunk 1]
```diff
@@ -33,50 +33,50 @@ Also please check the [Custom Component](https://www.highcharts.com/docs/dashboa
 ### Highcharts Component
 The option to include a Highcharts chart in one of the components is available out of the box. Here is the set of files that need to be included to make the Highcharts component work.
 ```html
-    <script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
-    <script src="https://code.highcharts.com/highcharts.js"></script>
-    <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
```
Comment (karolkolodziej) [inline docs/dashboards/types-of-components.md:36]: Why don't you remove those spaces in other `.md` files like `installation`,`edit-mode` etc

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: docs/dashboards/types-of-components.md [hunk 2]
```diff
@@ -33,50 +33,50 @@ Also please check the [Custom Component](https://www.highcharts.com/docs/dashboa
...
 ```
 
 Also the set of CSS styles needs to be imported, so that the DataGrid displays correctly.
 ```css
-    @import url("https://code.highcharts.com/css/datagrid.css");
+@import url("https://code.highcharts.com/css/datagrid.css");
 ```
 Then you need to specify the component type with `type: 'DataGrid'`.
-If you connect this component to the data connector, the content of the component will be automatically filled with data, and will allow the user to change the data in the data connector and automatically in all components that also are connected to this connector, by editing values in the cell. [Here is the example](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboards/demo/datagrid-sync)
+If you connect this component to the data connector, the content of the component will be automatically filled with data, and will allow the user to change the data in the data connector and automatically in all components that also are connected to this connector, by editing values in the cell. [Here is the example](http://utils.highcharts.local/samples/#view/dashboards/datagrid-component/datagrid-options)
```
Comment (karolkolodziej) [inline docs/dashboards/types-of-components.md:62]: Are we sure we would like to reverence to the local demo?

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

File: docs/dashboards/custom-component.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ Dashboards.board({
     }]
 });
 ```
-[The live example can be found here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboards/components/custom-component).
+[The live example can be found here](https://www.highcharts.com/samples/embed/dashboards/components/custom-component).
```
Comment (bre1470) [inline docs/dashboards/custom-component.md:82]: Comment: This demo looks not right.

File: docs/dashboards/edit-mode.md [hunk 4]
```diff
@@ -4,14 +4,14 @@ Edit mode
...
+        contextMenu: {
             enabled: true,
-            contextMenu: {
-                enabled: true,
-                items: ['editMode']
-            }
-        },
+            items: ['editMode']
+        }
+    },
```
Comment (bre1470) [inline docs/dashboards/edit-mode.md:14]: Indent needs to be reduced.

File: docs/dashboards/installation.md [hunk 5]
```diff
@@ -4,33 +4,33 @@ Installation
 ### Install via NPM
 One of the ways of adding the Highcharts Dashboards to your web page is via npm script. You can get all Dashboards functionality in the package, which can be installed using the following script:
 ```bash
-    npm install @highcharts/dashboards
+npm install @highcharts/dashboards
 ```
 Then import the package in your project:
 ``` JS
-    import Dashboards from '@highcharts/dashboards';
+import Dashboards from '@highcharts/dashboards';
```
Comment (bre1470) [inline docs/dashboards/installation.md:11]:
``` JavaScript
import * as Dashboards from '@highcharts/dashboards';
```

**Review by bre1470 — CHANGES_REQUESTED** [review]

File: docs/dashboards/custom-component.md (hunk 3 above)
Comment (pawellysy) [reply docs/dashboards/custom-component.md:82]: Why?

Comment (bre1470) [reply docs/dashboards/custom-component.md:82]: Screenshot 2023-07-07 at 09 33 41

Comment (pawellysy) [reply docs/dashboards/custom-component.md:82]: It looks fine on my computer 🤔 😓

**Review by bre1470 — APPROVED** [review]
Thanks! Looks good. 👍

**Pull request merged**